static/src/js/util/granules.js

```diff
@@ -98,6 +98,34 @@ export const prepareGranuleParams = (state, projectCollectionId) => {
   }
 }
 
+/**
+ * Translates the values returned from prepareGranuleParams to the camelCased keys that are expected in
+ * the granules.search() function
+ * @param {object} params - Params to be passed to the granules.search() function.
+ * @returns Parameters to be provided to the Granules request with camel cased keys
+ */
+export const buildGranuleSearchParams = (params) => {
```

Comment:
Does this serve a specific piece of functionality that we could rename the method to? `prepareGranuleParams` should be called right before we send a request, this seems to be an extra step that is also stripping out keys that could lead to confusion down the road when we don't know where keys are being stripped from.

Reply:
I just matched what we were doing for collections 🤷‍♂